src/Components/Categories.js

```diff
@@ -1,9 +1,30 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  CATEGORY,
+  selectAllCategories,
+} from '../redux/categories/categoriesSlice';
 
-const Categories = () => (
-  <div>
-    <button type="submit">Check status</button>
-  </div>
-);
+const Categories = () => {
+  const dispatch = useDispatch();
+
+  const categories = useSelector(selectAllCategories);
+
+  const handleCheckStatus = () => {
+    dispatch(
+      CATEGORY({
```

Comment:
- [ ] If you define action type in reducer as I gave a link to you. You do not have to define components.

Reply:
I really don't understand this part, However, this is a function for dispatching the category action which was earlier created in the categoriesSlice. Please see export and import from the categoriesSlice.js to where the action is called. The check status Btn is what calls this action which is handled by the slide. nothing was defined just a functional call.